Makefile

```diff
@@ -0,0 +1,5 @@
+push:
```

Comment:
@oridistor what about the dockerfile? why wasn't it packaged with it?

Reply:
We have a working dockerfile? I'll fix it if so.